main.py

```diff
@@ -294,7 +295,8 @@ def gen_code(kontaktdaten):
     two = 'IPY' + random.choice(code_chars)
     three = ''.join(random.choices(code_chars, k=4))
     random_code = f"{one}-{two}-{three}"
-    print(f"Für die Cookies-Generierung wird ein zufälliger Code verwendet ({random_code}).\n")
+    print(
+        f"Für die Cookies-Generierung wird ein zufälliger Code verwendet ({random_code}).\n")
```

Comment:
Unnötiger Diff, der zu einem Merge-Konflikt mit meinem PR führen wird. Besser Rückgängig machen:

```suggestion
    print(f"Für die Cookies-Generierung wird ein zufälliger Code verwendet ({random_code}).\n")
```